scripts/markdown/readme/languages/german.md

```diff
@@ -65,7 +65,7 @@ docker run -d --name nocodb -p 8080:8080 nocodb/nocodb:latest
 ### Verwenden von NPM
 
 ```
-npm create-nocodb-app
+npm install create-nocodb-app
```

Comment:
It should be `npx create-nocodb-app`

Reply:
i think it works both

Reply:
have now also added NPX

Reply:
Actually they are different. We want to guide users to create project in an interactive way. (See the first screenshot). Besides, it should follow the main documentation (EN version). 

Using `npx create-nocodb-app`

<img width="1308" alt="image" src="https://user-images.githubusercontent.com/35857179/158733486-0fb8cb9a-ea5d-441e-892b-79d5aa7fdd83.png">

Using `npm install create-nocodb-app`

<img width="1270" alt="image" src="https://user-images.githubusercontent.com/35857179/158733544-d9b86819-761f-4e9e-8b59-d3640691e450.png">